apis/kueue/v1alpha1/workload_webhook.go

```diff
@@ -75,17 +75,17 @@ func (r *Workload) ValidateDelete() error {
 }
 
 func ValidateWorkload(obj *Workload) field.ErrorList {
-	var allErrs field.ErrorList
-	specField := field.NewPath("spec")
-	podSetsField := specField.Child("podSets")
+	allErrs := field.ErrorList{}
```

Comment:
why change this? it's better to start with a nil array so that there are no unnecessary allocations.

Reply:
I can't remember why I change this, but it seems like unreasonable, reverted.